src/app/systelab-components/tabs/tabs.component.ts

```diff
@@ -1,11 +1,11 @@
-import { AfterContentInit, Component, ContentChildren, EventEmitter, Output, QueryList } from '@angular/core';
+import { AfterContentInit, Component, ContentChildren, EventEmitter, Input, Output, QueryList } from '@angular/core';
 import { TabComponent } from './tab.component';
 
 @Component({
 	selector: 'systelab-tabs',
 	template: `
-                <ul class="nav nav-tabs" role="tablist">
-                    <li class="nav-item" *ngFor="let tab of tabs" (click)="doSelectTab(tab)">
+                <ul [ngClass]="'nav nav-tabs' + (showTabBackground ? '' : ' hideTabBackground')" role="tablist">
```

Comment:
Use [class.hideTabBackground]="condition" and avoid ngClass for simple conditions